doc/tutorial/mutli_arch.hpp

```diff
@@ -97,20 +100,21 @@ suffix. For each suffixes, the corresponding options will be used from [OPTIONS]
 Optionally, you can pass an `INTERFACE` name to be used for compiling each of the targets.
 Optionally, the `QUIET` flag will remove error messages.
 
-Once called, `eve_build_variants`sets up a variable in the current scope named `eve_${NAME}_variants`
-that contains the list of target building the shared libraries. It can be used to setup dependencies.
+Once called, `eve_build_variants`sets up a variable in the current scope named
+`eve_${NAME}_variants` that contains the list of target building the shared libraries. It can be
+used to setup dependencies.
 
 Let's say we want to compile our kernel for SSE2, SSE4.1 and AVX2.
 The CMake code will be looking like:
 
 ```cmake
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.18)
 project(eve-multi-arch LANGUAGES CXX)
 
-find_package(eve)
+find_package(eve CONFIG REQIURED)
 
 add_library(setup INTERFACE)
-target_link_libraries(setup INTERFACE ${EVE_LIBRARIES})
+target_link_libraries(setup INTERFACE eve::eve)
 target_compile_options(setup INTERFACE "-O3")
```

Comment:
@jfalcou In these cases, would it be better to let the user select the optimization level through cmake's `CMAKE_BUILD_TYPE`?

Reply:
Maybe, I don't know. The build_variant system is made so that you can apply an INTERFACE to your builds which may be more precise than a blanket set of flags. You tell me.

Now, this can be addressed in another PR maybe?

Reply:
I'll just resolve this for now because it's only in a little tutorial and is very insignificant. We can open something later if it's worth it.